service/src/worker.rs

```diff
@@ -72,10 +72,22 @@ where
 
 		for p in peers.iter() {
 			// Todo: once the two direct servers are merged, remove this.
-			let url = worker_url_into_async_rpc_url(&p.url)?;
+			let url = match worker_url_into_async_rpc_url(&p.url) {
```

Comment:
That will be made obsolete in PR #595 - but it will be easy to remove the conflict, so I don't mind.

Reply:
I think this part in the gossiping loop has been made obsolete indeed, by more than one PR. So these changes here are not needed anymore.